toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -219,18 +219,19 @@ popd
 rm -rf file-5.40
 touch /logs/status_file_complete
 
-echo Readline-7.0
-tar xf readline-7.0.tar.gz
-pushd readline-7.0
+echo Readline-8.1
+tar xf readline-8.1.tar.gz
+pushd readline-8.1
 sed -i '/MV.*old/d' Makefile.in
 sed -i '/{OLDSUFF}/c:' support/shlib-install
 ./configure --prefix=/usr    \
             --disable-static \
-            --docdir=/usr/share/doc/readline-7.0
+            --with-curses    \
+            --docdir=/usr/share/doc/readline-8.1
 make SHLIB_LIBS="-L/tools/lib -lncursesw"
-make SHLIB_LIBS="-L/tools/lib -lncurses" install
+make SHLIB_LIBS="-L/tools/lib -lncursesw" install
```

Comment:
Is that separation intended?  first without install, and then with install?
make SHLIB_LIBS="-L/tools/lib -lncursesw"
make SHLIB_LIBS="-L/tools/lib -lncursesw" install

Reply:
Yes, this is intentional and based from: https://www.linuxfromscratch.org/lfs/downloads/11.0-systemd/LFS-BOOK-11.0-NOCHUNKS.html#ch-system-readline